server/badge/badge_test.go

```diff
@@ -451,6 +451,24 @@ func TestHandlerRevisionIsEnabledShortCommitSHA(t *testing.T) {
 	assert.Contains(t, response, "(abc)")
 }
 
+func TestHandlerRevisionIsEnabledMultipleSources(t *testing.T) {
+	app := testApp()
+	app.Status.OperationState.SyncResult.Revision = ""
+	app.Status.OperationState.SyncResult.Revisions = []string{"aa29b85", "cf41g63"}
+
+	settingsMgr := settings.NewSettingsManager(t.Context(), fake.NewClientset(argoCDCm(), argoCDSecret()), "default")
+	handler := NewHandler(appclientset.NewSimpleClientset(app), settingsMgr, "default", []string{})
+	req, err := http.NewRequest(http.MethodGet, "/api/badge?name=test-app&revision=true", nil)
```

Comment:
Might want to fix this if you want a review. `httpNoBody: http.NoBody should be preferred to the nil request body (gocritic)`
I'm not a maintainer but if I was I might skip reviewing PRs that don't pass all checks especially if I'm really busy